fe2-android/app/src/main/res/layout/social_media_home_fragment.xml

```diff
@@ -22,38 +22,14 @@
       android:layout_height="match_parent"
       android:layout_marginBottom="@dimen/bottom_navigation_view_height">
 
-      <LinearLayout
+      <ListView
+        android:id="@+id/chirps_list"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:orientation="vertical">
-
-        <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-          android:layout_width="match_parent"
-          android:layout_height="match_parent"
-          android:id="@+id/swipe_refresh_chirps">
-
-          <ListView
-            android:id="@+id/chirps_list"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:divider="@color/categoryTab"
-            android:dividerHeight="@dimen/social_media_divider_height"
-            tools:listitem="@layout/chirp_card" />
-        </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
-
-      </LinearLayout>
-
-      <ImageButton
-        android:id="@+id/social_media_send_fragment_button"
-        android:layout_width="@dimen/round_button_size"
-        android:layout_height="@dimen/round_button_size"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentBottom="true"
-        android:layout_marginEnd="@dimen/margin_top"
-        android:layout_marginBottom="@dimen/margin_top"
-        android:background="@drawable/round_button"
-        android:contentDescription="@string/send_button_desc"
-        android:src="@drawable/ic_social_media_send_foreground" />
```

Comment:
You should not have deleted that I think. This causes the build to fail.

Reply:
Fixed